core/pom.xml

```diff
@@ -81,6 +81,12 @@
             <artifactId>expiringmap</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
I don't think we want to add this as a strong dependency in `core`.  Either it should be a separate module, or the dependency should be `<optional>true</optional>`.  It should probably then register via a `JdbiPlugin`.

Reply:
Agree. I favor spinning up a `jdbi3-immutables` module.

Reply:
I don't use maven very often, but I will try that